src/test/java/com/serotonin/bacnet4j/npdu/ip/IpNetworkBuilderTest.java

```diff
@@ -46,4 +56,61 @@ public void withBroadcast19() {
         assertEquals("192.168.223.255", builder.getBroadcastAddress());
         assertEquals("255.255.224.0", builder.getSubnetMask());
     }
+
+    @Before public void initialize(){
+        Random rand = new Random();
+        interfaceDetails = IpNetworkUtils.getLocalInterfaceAddresses();
+        efaceInfo = interfaceDetails.get(rand.nextInt(interfaceDetails.size()));  

```

Comment:
Using a random real network interface makes this test not repeatable.

Reply:
Also, lets not use the name `efaceInfo`, variable names should be understandable. Use `interfaceInfo` instead.